modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java

```diff
@@ -180,7 +185,7 @@ public void createAPIWithWSDLEndpoint(String apiName, String username, String pa
         driver.findElement(By.xpath("(//input[@name='create-api'])[2]")).click();
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("startFromExistingSOAPEndpoint")));
         driver.findElement(By.id("wsdl-url")).clear();
-        driver.findElement(By.id("wsdl-url")).sendKeys("http://www.webservicex.net/geoipservice.asmx?WSDL");
+        driver.findElement(By.id("wsdl-url")).sendKeys("http://www.webservicex.com/globalweather.asmx?WSDL");
         driver.findElement(By.id("startFromExistingSOAPEndpoint")).click();
```

Comment:
This API is no longer available to use freely. So we may need different endpoint for this.